lib/util.js

```diff
@@ -131,6 +131,15 @@ const util = {
     util.run('ninja', ['-C', config.outputDir, config.buildTarget], options)
   },
 
+  lint: (options = {}) => {
+    console.log('linting ' + config.projects['brave-core'].dir  + '...')
+    options.cwd = config.projects['brave-core'].dir
+    options = mergeWithDefault(options)
+    // git cl format checks rietveld.server is set. Just set null.
```

Comment:
what is rietveld.server?

Reply:
I think it is code review site of chromium project.

Reply:
I'm confused about what `git cl` has to do with lint. We want to call `vendor/depot_tools/cpplint.py` in some kind of push hook

Reply:
I didn't read full `git_cl.py`, but it seems it uses cpplint.py for lint internally.
How about providing two kinds of formatting ways - `npm run lint` and git hook?

Reply:
`git cl` is utilities for chromium code review - https://commondatastorage.googleapis.com/chrome-infra-docs/flat/depot_tools/docs/html/git-cl.html
It has many subcommands and we can easily use `format` or `lint` subcommand for our formatting

Reply:
but there's no changelist to run it on. We need it to run on directory. I just tried to run and got `Cannot lint an empty CL`
we also need to make sure we're using cpplint.py in all cases

Reply:
Did you have any modification in `src/brave`?
That command tries to reformat files that has changes.

Reply:
lint should fail right now whether there are changes or not because I've seen dozens of what would be lint errors in the code

Reply:
If you use `format` it will detect the errors.
As I told you in DM, if we want to use `cpplint.py` instead of `clang_format`, `git cl` would not be the good choice.